src/intrinsics/mod.rs

```diff
@@ -206,10 +207,26 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 this.write_scalar(res, dest)?;
             }
 
+            "sqrtf32" => {
+                let [f] = check_arg_count(args)?;
```

Comment:
This function does not exist any more, it is now called `check_intrinsic_arg_count`